testing/agent-for-testing/build.gradle.kts

```diff
@@ -70,10 +70,6 @@ project(":instrumentation").subprojects {
 }
 
 tasks {
-  jar {
-    enabled = false
```

Comment:
Hmm, `jar` and `shadowJar` produce jars with the exact same name (since shadow jar has `archiveClassifier.set("")`). Is it safe to assume that the shadowed one will be published?